src/vault/client/api_util.clj

```diff
@@ -6,13 +6,29 @@
     [clojure.tools.logging :as log]
     [clojure.walk :as walk])
   (:import
+    (clojure.lang
+      ExceptionInfo)
     (java.security
       MessageDigest)
     (org.apache.commons.codec.binary
       Hex)))
 
 ;; ## API Utilities
 
+(defmacro support-not-found
```

Comment:
When reading how this macro was used, I found it difficult to tell where `opts` was being provided since it was below a largish expression. I have a slight preference to switch the order here, then be able to accept an arbitrary body as a vararg:

```clojure

(defmacro support-not-found
  [on-fail-opts & body]
  (try
    ~@body
    (catch ExceptionInfo ex#
       ,,,
```

Then called like:

```clojure
(api-util/support-not-found opts
  (let [response ,,,
```

Reply:
Makes sense 👍 will change!